docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-feature-integration.md

```diff
@@ -98,28 +99,60 @@ Verify that the following changes have been implemented by checking your databas
-| Item.productOfferReference         | attribute | created | src/Generated/Shared/Transfer/ItemTransfer                   |
+| ProductOfferStore                  | class     | created | src/Generated/Shared/Transfer/ProductOfferStoreTransfer           |
+| ProductOfferResponse               | class     | created | src/Generated/Shared/Transfer/ProductOfferResponseTransfer           |
+| ProductOfferError                  | class     | created | src/Generated/Shared/Transfer/ProductOfferErrorTransfer           |
+| ProductOfferCollection             | class     | created | src/Generated/Shared/Transfer/ProductOfferCollectionTransfer           |
+| ProductOfferCriteria               | class     | created | src/Generated/Shared/Transfer/ProductOfferCriteriaTransfer           |
 | ProductOfferValidity               | class     | created | src/Generated/Shared/Transfer/ProductOfferValidityTransfer   |
-| ProductOffer.productOfferValidity  | attribute | created | src/Generated/Shared/Transfer/ProductOfferTransfer           |
+| ProductOfferValidityCollection     | class     | created | src/Generated/Shared/Transfer/ProductOfferValidityCollectionTransfer   |
+| ProductPageSearch.merchantReferences | property  | created | src/Generated/Shared/Transfer/ProductPageSearchTransfer   |
+| ProductPayload.merchantReferences    | property  | created | src/Generated/Shared/Transfer/ProductPayloadTransfer   |
+| PageMap.merchantReferences           | property  | created | src/Generated/Shared/Transfer/PageMapTransfer   |
+| ProductAbstractMerchant.merchantReferences | property  | created | src/Generated/Shared/Transfer/ProductAbstractMerchantTransfer   |
+| ProductView.productOfferReference | property  | created | src/Generated/Shared/Transfer/ProductViewTransfer   |
+| ProductOfferStorage               | class  | created | src/Generated/Shared/Transfer/ProductOfferStorageTransfer   |
+| ProductOfferStorageCollection     | class  | created | src/Generated/Shared/Transfer/ProductOfferStorageCollectionTransfer   |
+| ProductOfferStorageCriteria     | class  | created | src/Generated/Shared/Transfer/ProductOfferStorageCriteriaTransfer   |
+| ProductStorageCriteria.merchantReference     | property  | created | src/Generated/Shared/Transfer/ProductStorageCriteriaTransfer   |
+
 
 {% endinfo_block %}
 
-### 3) Add Zed translations
+### 3) Add translations
+
+Append glossary according to your configuration:
+
+**src/data/import/glossary.csv**
+
+```csv
+product-offer.info.reference.invalid,"Product offer reference not found for product with SKU '%sku%'.",en_US
+product-offer.info.reference.invalid,"Produktangebotsreferenz für Produkt mit SKU '%sku%' nicht gefunden.",de_DE
+product-offer.message.not-active-or-approved,"Product offer not active for product with SKU '%sku%'.",en_US
+product-offer.message.not-active-or-approved,"Produktangebot ist inaktiv für Produkt mit SKU '%sku%'.",de_DE
```

Comment:
I have feeling that we have already added it in some PRs, no?

Reply:
seems, it should be only under Yves section only.. fixed